src/terminal/scroll/stack.lua

```diff
@@ -0,0 +1,64 @@
+-- Terminal Scroll Stack Module.
+-- Manages a stack of scroll regions for terminal control.
+-- @module scroll.stack
+local M = {}
+M.output = require("terminal.output")
+
+package.loaded["terminal.scroll.stack"] = M
+
+local _scroll_reset = "\27[r"
```

Comment:
this constant now is in 2 modules. Should we export it as a function, like the others, from `terminal.scroll` and reuse that one here? Then we have the constant only in 1 place.